coverage.c

```diff
@@ -653,6 +664,10 @@ int main_coverage(int argc, char *argv[]) {
         }
     }
 
+    if (print_value_warning) {
+        print_error("coverage", "Warning:  Missing quality values in alignments.  Mean base quality calculated only on available values.\n");
```

Comment:
I'm not sure if it's deliberate or not to add an extra line, but `print_error` already prints a newline.  It's hardly unique in this front though.  It looks like we have 18% of calls adding their own explicit newline.

Reply:
You know, I even checked what `print_error` did and didn't notice that.

Reply:
I did notice that nearly all other uses of `print_error` with a newline come from bam_markdup ;-)  [Guilty too of a couple in amplicon_stats]